### ISSUE TITLE
View Profile Status Kesehatan

### ISSUE THREAD

**Issue by yohang88** [jabardigitalservice/pikobar-flutter#93]
TBD.

**Comment by anandarh** [comment]
Popup pertanyaan setiap hari? Nanti hasil dari pertanyaan ini dievaluasi pihak berwenang untuk mengupdate status kesehatan usernya? Apa tidak akan kewalahan untuk melakukan pemeriksaan dari banyak usernya? Mas @yohang88

**Comment by yohang88** [comment]
@anandarh Yup, mekanismenya masih panjang diskusinya, siapa dan bagaimana cara update status kesehatan ini euy. Perlu "engine" lain yang melakukan update ke record status ini.

Sementara kita siapkan aja fieldnya di masing-masing user (lihat userku) = `health_status` dan `health_status_label`

Dan tampilan di view profile tolong disiapkan pakai Remote Config aja, sementara kita disabled agar tidak tampil dulu field Status Kesehatannya.

**Comment by anandarh** [comment]
@yohang88 Mungkin bisa seperti ini dulu.

- User diberi pertanyaan 1x tiap hari. 
Pertanyaan itu dapat memberi nilai probabilitas keadaan user tersebut, misal dari 1-10, dimana 1 adalah user sehat dan 10 adalah kemungkinan terbesar dia positif.

- Jawaban dan nilai probabilitas itu direcord setiap hari.

- Setiap 7 hari (seminggu), nilai probabilitas itu dikalkulasikan (by sistem) untuk menentukan status kesehatannya. 
Misal 7 hari berturut-turut, user tersebut mendapat nilai probabilitas 9, maka secara otomatis sistem akan mengupdate status kesehatannya menjadi merah.

Contoh pertanyaan & aplikasinya (Aplikasi kontribusi dari Prodi Teknik Informatika , Universitas Buana Perjuangan Karawang).

![WhatsApp Image 2020-03-24 at 16 50 37](https://user-images.githubusercontent.com/26994065/77749073-fd26db00-7053-11ea-829c-49d781e916f7.jpeg)

[SCCovid-19Release.apk.zip](https://github.com/jabardigitalservice/pikobar-flutter/files/4392936/SCCovid-19Release.apk.zip)

**Comment by adtrimurdani** [comment]
Kalo pertanyaan2 nyambungnya ke Prixa bisa ga ya? Mereka kan punya scoring utk menentukan health status pengguna.

Kemudian apakah fitur Status Kesehatan ini jugs bisa digunakan untuk menentukan sehat, ODP, PDP, atau positif? Jika Status Kesehatan sampai bisa menentukan sampai kearah sana, maka data ini akan digunakan untuk visuailisasi pergerakan pada user transmission dashboard yg didevelop Geocreate itu.

@yohang88 @anandarh

**Comment by denisenricohasyim93** [comment]
Punten Braders ... @yohang88 @anandarh @adtrimurdani dan teman-teman berikutnya ...
Mohon ijin ...
Berikut Scheduled Pikobar Public Health Check Monitoring and Analysis Proposed Solution dari Kami ... Mangga di sinau bareng ...
Hatur Nuhun ... 
🙏

![Scheduled Pikobar Public Health Check Monitoring and Analysis Proposed Solution](https://user-images.githubusercontent.com/26061493/78460360-68268080-76ea-11ea-8b6e-845c072928e0.jpg)
![Scheduled Pikobar Public Health Check Monitoring and Analysis Proposed Solution (1)](https://user-images.githubusercontent.com/26061493/78460362-6e1c6180-76ea-11ea-84c9-de8b1241f332.jpg)
![Scheduled Pikobar Public Health Check Monitoring and Analysis Proposed Solution (2)](https://user-images.githubusercontent.com/26061493/78460366-7379ac00-76ea-11ea-8351-c5e44dd86aab.jpg)
[Scheduled Pikobar Public Health Check Monitoring and Analysis Proposed Solution.pdf](https://github.com/jabardigitalservice/pikobar-flutter/files/4432351/Scheduled.Pikobar.Public.Health.Check.Monitoring.and.Analysis.Proposed.Solution.pdf)

**Comment by yohang88** [comment]
Terima kasih banyak mas @denisenricohasyim93 atas ide dan masukannya, serta rekan-rekan lain.

Fyi saat ini data kasus/pasien dikelola pada aplikasi Pelaporan Kasus internal di PIKOBAR, dan data tersebut dientry oleh Admin masing-masing Dinkes Kab/Kota. 

Masih dibahas, karena status ODP/PDP/Positif saat ini harus dideclare/diset oleh Dinkes Kab/Kota, dan tidak bisa self-assessment. Jadi nampaknya memang harus ada integrasi dengan aplikasi pelaporan. 

ASAP segera diupdate di sini.
